conf/homeserver.yaml

```diff
@@ -2331,7 +2331,7 @@ email:
   # to the identity server as the org.matrix.web_client_location key. Defaults
   # to unset, giving no guidance to the identity server.
   #
-  #invite_client_location: https://app.element.io
+  invite_client_location: __INVITE_CLIENT_LOCATION__
```

Comment:
Maybe is not better use the same url than the element url that we used above ?

Reply:
I would keep 3 different values for Client because they enable different things.
I tried to have only one `__ELEMENT_YNH_URL__` but we can only bind one setting through the config panel. Workaround would be to have a custom getter/setter to set the value in all 3 places.

Also to unset guidance, we may need to comment out the line entirely. If yes, @thardev could contribute a `getter/setter` with some `sed`

Reply:
or maybe we should set different client values depending on if the server is configured with open registration or closed with SSO. See /install lines 89-102

Reply:
solved thanks to @rosbeef